spec/hosts/nodes/auxtel-dc01.cp.lsst.org_spec.rb

```diff
@@ -4,28 +4,50 @@
 
 describe 'auxtel-dc01.cp.lsst.org', :sitepp do
   on_supported_os.each do |os, os_facts|
+    next unless os =~ %r{almalinux-9-x86_64}
+
     context "on #{os}" do
       let(:facts) do
-        lsst_override_facts(os_facts,
-                            is_virtual: false,
-                            virtual: 'physical',
-                            dmi: {
-                              'product' => {
-                                'name' => 'PowerEdge R630',
-                              },
-                            })
+        override_facts(os_facts,
+                       fqdn: 'auxtel-dc01.cp.lsst.org',
+                       is_virtual: false,
+                       virtual: 'physical',
+                       dmi: {
+                         'product' => {
+                           'name' => 'PowerEdge R630',
+                         },
+                       })
       end
       let(:node_params) do
         {
           role: 'atsccs',
```

Comment:
This role is incorrect... I have this fixed in another PR but this one will probably go in first.

Reply:
Oh i just used the same role that was in foreman.